web-client/public/js/upload.js

```diff
@@ -102,18 +102,24 @@ $(function () {
   //normalization stuff
 
   var normalization = false;
+  var grayThreshold = false;
 
   $("#normalization-button").click(function(){
     normalization = true;
     displayNetwork(currentNetwork, name, normalization);
   });
 
+  $('#grayThresholdInput').on("change", function(){
+    grayThreshold = true;
+    displayNetwork(currentNetwork, name, normalization, grayThreshold);
+  });
 
-  var displayNetwork = function (network, name, normalization) {
+  var displayNetwork = function (network, name, normalization, grayThreshold) {
 
     currentNetwork = network;
     console.log(network); // Display the network in the console
     console.log(normalization);
```

Comment:
We might want to be leaving more informative console.logs, if we are going to be including them at all. Even just a console.log("normalization: " + normalization); would be sufficient